### PR TITLE
WIP: Add support to gather cgroup cpuacct percpu usage

### PR DESCRIPTION
**Pull request by itkovian**

### CODE REVIEW

**Comment by itkovian** [comment]
Look into using the new SPACE plugin to get this data out
- talk to rabbitMQ or Kafka?
- is this in the job record?

**Comment by wdpypere** [comment]
@boegel @stdweird my C is not good enough to review this, can you have a look?

**Comment by boegel** [comment]
@wdpypere me == clueless, this is for @stdweird imhop

**Comment by wdpypere** [comment]
@stdweird can you review this?

File: src/plugins/jobacct_gather/cgroup/jobacct_gather_cgroup.c [hunk 1]
```diff
@@ -102,8 +102,10 @@ static slurm_cgroup_conf_t slurm_cgroup_conf;
 static void _prec_extra(jag_prec_t *prec)
 {
 	unsigned long utime, stime, total_rss, total_pgpgin;
-	char *cpu_time = NULL, *memory_stat = NULL, *ptr;
-	size_t cpu_time_size = 0, memory_stat_size = 0;
+	char *cpu_time = NULL, *memory_stat = NULL, *cpuacct_percpu = NULL, *ptr;
+	size_t cpu_time_size = 0, memory_stat_size = 0, cpuacct_percpu_size = 0;
+    int i = 0i, pos = 0;
```
Comment (stdweird) [inline src/plugins/jobacct_gather/cgroup/jobacct_gather_cgroup.c]: if you want this upstream, you'll have to fix the indentation.

**Comment by stdweird** [comment]
@itkovian is there a slurm support ticket for this?

**Comment by wdpypere** [comment]
@itkovian where should this PR find its future? Are we doing somting on this? Should it be updated? closed? put on hold?

**Comment by itkovian** [comment]
I'm going to close it and try to make an effort to get this going in 20.02 again.

**Pull request closed without merging**